python/external_config/external_command.py

```diff
@@ -237,6 +243,13 @@ def display_name(self):
         """
         return self._display_name
 
+    @property
+    def icon(self):
+        """
+        The path to a square png icon file representing this item
```

Comment:
we should add : "or None if no icon is associated"

Reply:
Should the property be called icon_256 ? for consistency with overall TK interface ?

Reply:
Not sure whether it's None, I really just copy over what I get from the `data` dict, I would assume it might be empty string

Reply:
I'd prefer to stick with "icon" instead of "icon_256"

Reply:
My guess would be a `None` and not an empty string, but I'm not 100% sure on that.